instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import CommentSection from '../CommentSection/CommentSection.js';
+import Post from './Post.js';
+import './PostContainer.css';
+import styled from 'styled-components';
+
+const Cont = styled.div`
+width: 85%;
+margin: 0 auto;
+`
+const CommentContainer = styled.div`
+    width: 85%;
+    margin: 0 auto;
+`
+class PostContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            likes: this.props.data.likes,
+        }
+    }
+
+    handleLikeClick = () => {
+        this.setState({
+            likes: this.state.likes + 1
+        })
```

Comment:
With this function as it is, I believe likes can go up infinitely, try adding toggle functionality sometime!